### PR TITLE
Add Kibana dashboard test

### PR DESCRIPTION
**Pull request by branden**
**What type of PR is this?**

Chore

**What this PR does/ why we need it**:

This adds a test which asserts that expected Kibana dashboards are present.

**Which issue(s) this PR fixes**:

* https://jira.d2iq.com/browse/D2IQ-71039

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Checklist**

* [x] The commit message explains the changes and why are needed.
* [x] The code builds and passes lint/style checks locally.
* [x] The relevant subset of integration tests pass locally.
* [x] The core changes are covered by tests.
* [x] The documentation is updated where needed.

### CODE REVIEW

File: test/run-tests.sh [hunk 1]
```diff
@@ -17,7 +17,8 @@ echo "INFO: the following test groups will be run:"
...
 
-for g in ${TESTS[*]}
-do
-    go test -tags experimental -timeout 60m -race -v -run "$g"
-done
+#for g in ${TESTS[*]}
+#do
+#    go test -tags experimental -timeout 60m -race -v -run "$g"
+#done
+go test -tags experimental -timeout 60m -race -v -run "TestElasticsearchGroup"
```
Comment (shaneutt) [inline test/run-tests.sh:24]: It would appear this was done for local debugging and this needs to be put back :thinking:

**Review by shaneutt — CHANGES_REQUESTED** [review]

File: test/run-tests.sh (hunk 1 above)
Comment (branden) [reply test/run-tests.sh:24]: Oh jeeze, yup. Thanks!

**Review by shaneutt — APPROVED** [review]
LGTM :heavy_check_mark:

**Review by joejulian — APPROVED** [review]

**Review by samvantran — APPROVED** [review]
👍 do we need to be concerned with the two "required" tests that haven't started? all the other tests passed

**Comment by branden** [comment]
Huh not sure what happened with those checks. I gave kubeaddons-tests a kick. I think this is the continuous-integration/teamcity job from my last push, still running: https://teamcity.mesosphere.io/buildConfiguration/kubeaddons_KubernetesBaseAddons/2872529 I expect that'll report its status when it finishes.

**Comment by branden** [comment]
Passing CI run here: https://teamcity.mesosphere.io/buildConfiguration/kubeaddons_KubernetesBaseAddons/2873058

If TC were reporting statuses correctly then all checks would be green, so I'll merge.

**Pull request merged**